### PR TITLE
Use Bootstrap responsive grid

### PR DESCRIPTION
**Pull request by crgc**
- Added basic skeleton
- Styled header nav bar and footer
- Added Bootstrap classes for small, medium, and large screens

### CODE REVIEW

File: resources/css/index.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.header-links-categories ul {
+  margin-bottom: 0;
+}
+
+.header-links-categories ul li a {
+  color: black;
+  text-decoration: none;
+}
```
Comment (ghost) [inline resources/css/index.css]: Avoid using custom styling where bootstrap styling is possible. Example in the above, you can text-decoration-none and text-dark bootstrap classes. Please check the rest of your styling as well.

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉

## Hello team 👋,

**Great job on this milestone 👏.**

## Highlights.

- **You have installed Bootstrap properly.**
- **The font colors and sizes of elements match those of the elements in the original page.**

## Improvements to note moving forward ▶️ 
- **Ensure your webpage is fully responsive. Example, the menu at the bottom of the red navbar hides and a hamburger button is seen for medium and small screen sizes; the login button changes to an icon for small-screen sizes.**
- **Remember to add the input field at the bottom of the red navbar as it is shown in the original page.**
- **The login and subscribe buttons should be links. The content below the Newsweek logo at the footer section should be links as well.**
- **Fix the linter errors.**
- **Check more in the comment below.**

## Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Review by ghost — COMMENTED** [review]
# Status: Invalid

**Hello team, I marked this milestone as invalid because it doesn't have a descriptive README file. You can use the Microverse readme [template](https://github.com/microverseinc/readme-template) as a guideline.**

**Submit for another review after you have implemented the above change.**

## Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Comment by LoxiasCam** [comment]
### Using Boostrap: Milestone 2

Added three responsive column sections for:

- Featured stories and Sponsored insights
- Top Story and other stories
- The Debate and further sections
- Fixed linter errors

Also fixed some styling issues in the navbar and footer, and implemented the use of more Bootstrap classes to avoid using custom styling.

File: resources/css/index.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+.header-links-trending ul {
+  font-size: 17px;
+  font-size: 16px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-align: center;
+  margin-bottom: 0;
+}
```
Comment (ghost) [inline resources/css/index.css:74]: Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-uppercase`, `text-center`, `mb-0` bootstrap classes. Please check the rest of your styling as well.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

## Hello team 👋,

**Great job on this milestone 👏.**

## Highlights

- **Your webpage is responsive.**
- **Your linter checks are working properly.**

## Improvements to note moving forward ▶️ 

### Desktop screen size

- **Column one should have Sponsored insights / Rankings section below the Featured stories section.**
- **Column two should have the Latest News section below the More Stories section.**
- **The design for The Debate section, in column three is incomplete. Also, column three should have Opinion, Sponsored insights / Rankings and Free signup sections.**

### Mobile screen size

- **Fix the order in which of your sections display in the mobile screen size so it matches that of the original page. For example, the sections following Top Story should Featured Stories, The Debate... In your page, the sections after Top Story are My Turn, Cultural & Travel...**

### General
- **Images in Featured Stories, Top Story, Cultural & Travel and Sponsored sections should have labels at the bottom leftmost edge of every image. In your case, a label appears at the bottom of the image.**
- **All images and titles should be links.**
- **Remove the page horizontal scroll.**
- **Check one more in the comment below.**

## Happy coding 🥂

**Pull request merged**